build/operator/Dockerfile

```diff
@@ -1,4 +1,24 @@
-FROM quay.io/centos/centos:stream9
+ARG BUILD_ARCH=amd64
+FROM --platform=linux/${BUILD_ARCH} quay.io/centos/centos:stream9 AS builder
+
+RUN dnf install -y tar gzip jq && dnf clean all
+RUN ARCH=$(uname -m | sed 's/x86_64/amd64/') && \
+    GO_VERSION=$(curl -L -s "https://go.dev/dl/?mode=json" | jq -r '.[0].version') && \
```

Comment:
i dont think we want unpinned go version?
stuff can break suddenly

unless we are fine with that or if this was the case already, lets see other maintainers idea

Reply:
This was already the case; this PR follows the same approach.

Reply:
@oshoval For this as well, I will create a separate issue, resolve it, and provide a follow-up.

Reply:
thanks, if this was already the case, no rush to fix it, its ok

Reply:
Please consider in the follow-up to use ADD if it helps